app.js

```diff
@@ -1,5 +1,50 @@
-// import functions and grab DOM elements
+const generateNumber = Math.floor(Math.random() * 20);
+const numberChecker = document.getElementById('numberChecker');
+const userInput = document.getElementById('input');
+const output = document.getElementById('output');
+const reset = document.getElementById('reset');
 
-// initialize state
+console.log(generateNumber);
 
-// set event listeners to update state and DOM
\ No newline at end of file
+let guessesLeft = 4;
+
+numberChecker.addEventListener('click', () => {
```

Comment:
Looks good, if you look over the class rubric, you can see that this assignment is also looking for a compareNumbers function as well as having it tested.  I think you will need to create that logic and apply it to the appropriate parts of the eventlistener.